.github/workflows/ci.yml

```diff
@@ -29,85 +29,61 @@ on:
 
 jobs:
   build:
-    runs-on: ${{ matrix.runner || 'ubuntu-22.04' }}
-    container: ${{ matrix.container }}
-    name: ${{ matrix.name }}
     strategy:
       fail-fast: false
       matrix:
+        lib_type: [shared, static]
+        # Can't test DB2 as required db2exc package is no longer available after Ubuntu 14.04
+        backend: [SQLite3, PostgreSQL, MySQL, Firebird, Oracle, ODBC, Empty, Valgrind]
+        runner: [ubuntu-22.04]
         include:
-          # Note: the jobs are ordered in the order of decreasing running
-          # time, as this should minimize the total run-time of all jobs.
-          - backend: postgresql
+          - lib_type: shared
+            backend: Empty
             runner: macos-14
-            name: PostgreSQL macOS
-          - backend: oracle
-            name: Oracle 11
             no_boost: true
-          - backend: valgrind
-            name: Valgrind
-          - backend: odbc
-            # There are many leak reports under Ubuntu 22.04, see #1008.
```

Comment:
Has #1008 been solved too?

Reply:
I didn't do anything about it in particular but there are no errors on the CI, so I guess yes? :eyes: